workflow_imaging/ingest.py

```diff
@@ -1,65 +1,73 @@
 import scanreader
+import numpy as np
+import pandas as pd
```

Comment:
`pandas` is only used for reading a csv into a list of dicts, which can be accomplished with the `csv` module in the Python standard library. To reduce dependencies and increase stability, use the most standard solution: https://docs.python.org/3/library/csv.html#csv.DictReader